### PR TITLE
chore: use rsetup instead of manual configs

### PR DESCRIPTION
**Pull request by RadxaPanda**

### CODE REVIEW

**Comment by RadxaPanda** [comment]
@RadxaYuntian pls check this out

**Comment by RadxaYuntian** [comment]
You can request reviewer on the right side. There should also be a CODEOWNERS file to automate this step.

File: debian/postinst [hunk 1]
```diff
@@ -7,15 +7,11 @@ if [ "$1" = "configure" ]; then
...
-    fi
-
-    if [ -e /etc/gdm3/daemon.conf ]; then
-        sed -i s/"\[daemon\]"/"\[daemon\]\nAutomaticLogin=oem\nAutomaticLoginEnable=True"/g /etc/gdm3/daemon.conf
-    fi
+    rsetup set_autologin_status lightdm.service oem on
+    rsetup set_autologin_status gdm.service oem on
+    rsetup set_autologin_status sddm.service oem on
+    rsetup set_autologin_status getty@.service oem on
+    rsetup set_autologin_status serial-getty@.service on
```
Comment (RadxaYuntian) [inline debian/postinst]:
```suggestion
    rsetup set_autologin_status serial-getty@.service oem on
```

File: debian/postrm [hunk 2]
```diff
@@ -6,6 +6,11 @@ set -e
 case $1 in
     remove)
         # Lock the user
+        rsetup set_autologin_status lightdm.service oem off
+        rsetup set_autologin_status gdm.service oem off
+        rsetup set_autologin_status sddm.service oem off
+        rsetup set_autologin_status getty@.service oem off
+        rsetup set_autologin_status serial-getty@.service off
```
Comment (RadxaYuntian) [inline debian/postrm]:
```suggestion
        rsetup set_autologin_status serial-getty@.service oem off
```

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

File: debian/control [hunk 3]
```diff
@@ -12,5 +12,7 @@ Architecture: all
 Section: admin
 Priority: standard
 Depends: ${misc:Depends},
+         rsetup,
+         systemd,
```
Comment (RadxaYuntian) [inline debian/control]:
```suggestion
```
If we ever support multiple init systems, it should be `rsetup` to handle the difference between them.

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
squash

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**